HUD/SetMinimapComponent.md

```diff
@@ -9,14 +9,26 @@ Any SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int hudColor);
 ```
 
 This native is used to colorize certain map components like the army base at the top of the map.
+Colors are being mixed, so for example using blue (HudColor: 9) on yellow lift-off markers
```

Comment:
"The provided color will be mixed with the existing component color, for example" might be better wording and provide more context